index.js

```diff
@@ -346,7 +347,13 @@ module.exports = {
   },
 
   postBuild(result) {
-    this._updateAppFilesInManifest(result.directory);
+    // Need to update manifest from index.html.
+    // Set environment variable `UPDATE_MANIFEST_FROM_HTML` to `true`
+    // Usage `UPDATE_MANIFEST_FROM_HTML=true ember s`
+    if(process.env.FASTBOOT_UPDATE_MANIFEST_FROM_HTML === 'true') {
```

Comment:
When will this be set to true? Is it only for embroider builds that this flag is true?

Reply:
@kratiahuja this will also be true for non embroider builds using this new "treat index.html" as the entry point feature.  If this feature turns out to work well, we would like to explore it being the default.

Reply:
YEah I think that would definitely help people understand the changes. I think we probably also need to update the readme for the same so folks can opt-in.

Reply:
Added a reed me section for this flag.